src/main/java/com/ceritoxi/euler5/Euler5.java

```diff
@@ -0,0 +1,21 @@
+package com.ceritoxi.euler5;
+
+import com.ceritoxi.util.Math;
+
+public class Euler5 {
+	
+	private int result;
+	
+	public Euler5() {
+		int[] argument = {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20};
```

Comment:
```arguments```

We always name variables in plural form (if possible) when they mean a collection of any kind.